src/structs/field-shorthand.md

```diff
@@ -32,6 +32,20 @@ impl Person {
         Self { name, age }
     }
 }
+```    
+* Implement a `default` setting for the struct.
+```rust,ignore
```

Comment:
Why do you have the `ignore` attribute on the code here? It will make `mdbook test` [ignore](https://rust-lang.github.io/mdBook/format/mdbook.html) the code — but we actually want to compile it so that we know it works.